bin/katello-certs-check

```diff
@@ -39,14 +42,17 @@ done
 
 EXIT_CODE=0
 
-if [ -z "$CERT_FILE" -o -z "$KEY_FILE" -o -z "$CA_BUNDLE_FILE" ]; then
+if [ -z "$CERT_FILE" -o -z "$KEY_FILE" -o -z "$CA_BUNDLE_FILE" -o -z "$TARGET" ]; then
     echo 'One of the required parameters is missing.' >&2
     usage
     exit 1
 fi
 
-HOSTNAME=$(hostname -f)
-CERT_HOSTNAME=$(openssl x509 -noout -subject -in $CERT_FILE | sed -e 's/^subject.*CN=\([a-zA-Z0-9\.\-]*\).*$/\1/')
+if [[ $TARGET != "foreman" && $TARGET != "foreman-proxy" ]]; then
+    echo -e 'Target not specified or incorrect. Target value should be foreman or foreman-proxy \n' >&2
```

Comment:
For the error, I think we should have Foreman and Foreman-proxy capitalized and then take the input from the user and make it lowercase or adjust for that, but that can be done in a separate pr and is not blocking this, more just a grammar nitpick.